.travis.yml

```diff
@@ -7,6 +7,8 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
+git:
+  depth: false
```

Comment:
This would cause travis to clone the entire repository with all the commits right ? 
Would this slow down the travis build ?